styles/tokens/color.json

```diff
@@ -230,6 +230,16 @@
       },
       "file": "color"
     },
+    "15": {
```

Comment:
Jane does not want to add this to colors page. It will show up on tokens page of course.